### ISSUE TITLE
Left Aligned History and Help obscure JSON Path

### ISSUE THREAD

**Issue by brent-hoover** [getinsomnia/insomnia#7421]
### Expected Behavior

Should be able to see the JSON path value

### Actual Behavior

Cannot see it as it obscured by a left-aligned Help and History regardless of how wide you make the app or even when you click in to modify

![image](https://github.com/Kong/insomnia/assets/72819/85651725-4e61-44dc-bf1a-23e1e1dcb2a7)

### Reproduction Steps

1. Open the app on Mac OS, Insomnia version 9.2
2. Observe the JSON Path value is obscured

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

Version: Insomnia 9.2.0 Build date: 5/15/2024 OS: Darwin arm64 23.4.0 Electron: 29.3.0 Node: 20.9.0 Node ABI: 121 V8: 12.2.281.27-electron.0 Architecture: arm64

### What operating system are you using?

macOS

### Operating System Version

Sonoma 14.4.1

### Installation method

Download from website

### Last Known Working Insomnia version

I don't know exactly which version but this bug was introduced pretty recently

**Comment by m-atthart** [comment]
It was introduced in 9.2 beta 0. I was on 9.1.1 and did not have the issue until I updated. I also downloaded 9.1.1 and 9.2 beta 0 from the releases page to verify. I can also confirm that the issue exists on Windows.

I can't get the repo to run locally, so I can't know for sure, but my first guess is it's from code-editor.tsx in [this commit](https://github.com/Kong/insomnia/commit/9b068f642ec48ed741e4130c9f6bdf5d2276cd25#diff-2cfa5f1135442d4e3c6548e006b81b08bdbb8393b48ae8609c733c6e4fcc0f15)

The search bar at the top of the response panel also seems to have a similar issue, but it has been like this since at least 8.6 (I just checked)
<img width="696" alt="Screenshot 2024-05-18 at 10 06 59" src="https://github.com/Kong/insomnia/assets/7551390/eaa525d5-191a-4415-8f71-9c251e912e16">
